### PR TITLE
Changes from mpes workshop

### PR DESCRIPTION
**Pull request by domna**

### CODE REVIEW

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 1]
```diff
@@ -5,19 +5,24 @@
 
 doc: This is the most general application definition for multidimensional photoelectron spectroscopy.
 category: application
-NXmpes:
+    NXmpes:
```
Comment (lukaspie) [inline contributed_definitions/nyaml/NXmpes.yaml:8]: This has been indented accidentally I guess.

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 2]
```diff
@@ -5,19 +5,24 @@
...
   (NXentry):
     title:
+      exists: recommended
     start_time(NX_DATE_TIME):
       doc: "Datetime of the start of the measurement."
+    end_time(NX_DATE_TIME):
+      exists: recommended
```
Comment (lukaspie) [inline contributed_definitions/nyaml/NXmpes.yaml:15]: see also my comments in the NXmpes_xps PR:
https://github.com/FAIRmat-NFDI/nexus_definitions/pull/30#discussion_r1192707961

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 3]
```diff
@@ -67,7 +84,8 @@ NXmpes:
           exists: recommended
           unit: NX_ANY
       (NXelectronanalyser):
-        description:
+        model(NXfabrication):
```
Comment (lukaspie) [inline contributed_definitions/nyaml/NXmpes.yaml:87]: Maybe manufacturer, model, and serial number could be recommended here.

File: contributed_definitions/nyaml/NXmpes.yaml (hunk 1 above)
Comment (domna) [reply contributed_definitions/nyaml/NXmpes.yaml]: Yes. Thanks for spotting it.

File: contributed_definitions/nyaml/NXmpes.yaml [hunk 4]
```diff
@@ -8,16 +8,21 @@ category: application
...
+      exists: recommended
     start_time(NX_DATE_TIME):
       doc: "Datetime of the start of the measurement."
+    end_time(NX_DATE_TIME):
+      exists: recommended
     definition:
       \@version:
-      enumeration: ["NXmpes"]
+      enumeration: ["NXmpes"] # ISO 18115
+    method: # ISO 18115
```
Comment (sanbrock) [inline contributed_definitions/nyaml/NXmpes.yaml:19]: please add a proper doc string

**Review by sanbrock — CHANGES_REQUESTED** [review]
add doc string to ISO based method

**Comment by domna** [comment]
Closing this in favour of #52

**Pull request closed without merging**